tests/Unit/Evolution/DiscontinuousGalerkin/SlopeLimiters/Test_WenoHelpers.cpp

```diff
@@ -82,11 +97,215 @@ void test_oscillation_indicator_3d() noexcept {
   CHECK(indicator == approx(expected));
 }
 
+struct ScalarTag : db::SimpleTag {
+  using type = Scalar<DataVector>;
+  static std::string name() noexcept { return "Scalar"; }
+};
+
+template <size_t VolumeDim>
+struct VectorTag : db::SimpleTag {
+  using type = tnsr::I<DataVector, VolumeDim>;
+  static std::string name() noexcept { return "Vector"; }
+};
+
+void test_reconstruction_1d() noexcept {
+  const double neighbor_linear_weight = 0.001;
```

Comment:
Is there a good reason not to test with another value of `neighbor_linear_weight`?

Reply:
I've changed the 1D case to use a different weight.